test/transformation-test.coffee

```diff
@@ -35,7 +35,11 @@ parseApiBlueprint = (source, type, cb) ->
 
     sourcemap = result?.sourcemap
 
-    err = adapter.transformError(source, err)
+    if err
+      # Protagonist does not include error in parse result, only in err
+      result.error = err
```

Comment:
can we be sure `result` is always an object when `err` is present?
(trying to avoid `TypeError: cannot set error of null`.